pkg/controller/node/node_controller.go

```diff
@@ -131,25 +147,19 @@ func (r *ReconcileNode) Reconcile(request reconcile.Request) (reconcile.Result,
 		return reconcileResult, nil
 	}
 
-	return reconcileResult, nil
+	return reconcile.Result{}, nil
```

Comment:
A successful reconciliation need not return a periodic update in case of node controller.